fdbserver/SimulatedCluster.actor.cpp

```diff
@@ -1376,6 +1379,10 @@ void checkExtraDB(const char *testFile, int &extraDB, int &minimumReplication, i
 		if (attrib == "minimumRegions") {
 			sscanf( value.c_str(), "%d", &minimumRegions );
 		}
+
+		if (attrib == "fearless") {
```

Comment:
Please use multi-region and not fearless

Reply:
removed the entire commit that introduced the fearless, because the feature works without disabling fearless.